### PR TITLE
Make InputListItem reuse ConfigurationWell, so passwords are masked.

### LINKED ISSUE #2198: 2.0.0 Local GELF TCP Input Displaying Password in ClearText
### Problem description

Viewing local inputs on 2.0.0, GELF TCP input is displaying tl_key_password in cleartext. While previously in 1.3 the password was correctly displayed as ******

### Steps to reproduce the problem

1. Menu: System -> Nodes : More Actions - Local Message Inputs
2. View an existing GELF TCP input.

### Environment

* Graylog Version: Graylog 2.0.0 (2dc6c03)
* Elasticsearch Version: 2.3.2
* MongoDB Version: 3.0.11
* Operating System: Ubuntu 14.04 - Linux 3.13.0-85-generic #129-Ubuntu SMP Thu Mar 17 20:50:15 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
* Browser version: Chrome Version 50.0.2661.94 m

### PR DESCRIPTION
**Pull request by dennisoelkers**
Before this change, the `InputListItem` component was creating the configuration list on its own, so passwords were not masked.

Fixes #2198.

### CODE REVIEW

File: graylog2-web-interface/src/components/inputs/InputListItem.jsx [hunk 1]
```diff
@@ -139,22 +140,21 @@ const InputListItem = React.createClass({
...
                    titleValue={input.title}
                    typeName={input.type} includeTitleField
                    submitAction={this._updateInput} values={input.attributes} /> : null;
 
     const additionalContent = (
       <div>
         <Col md={8}>
-          <Well bsSize="small" className="configuration-well">
-            <ul>
-              {this._getConfigurationOptions(this.props.input.attributes)}
```
Comment (edmundoa) [inline graylog2-web-interface/src/components/inputs/InputListItem.jsx]: We should also remove the `_getConfigurationOptions` method, as it is not used any more.

**Comment by edmundoa** [comment]
Other than the clean-up remark, it looks good to me! 👍

File: graylog2-web-interface/src/components/inputs/InputListItem.jsx (hunk 1 above)
Comment (dennisoelkers) [inline graylog2-web-interface/src/components/inputs/InputListItem.jsx]: ✅

**Comment by edmundoa** [comment]
Looking good! :shipit:

**Pull request merged**